index.html

```diff
@@ -2128,6 +2291,220 @@ <h4>Bidirectional text</h4>
+        <p>Since it is a requirement of DAPT that <a>DAPT Documents</a> include a
+        <code>ttp:contentProfiles</code> attribute on the root element,
+        and that the attribute includes a DAPT <a>content profile</a> designator,
+        as specified at <a href="#ttp-contentprofiles"></a>,
+        it follows that any TTML document that does not include such an attribute,
+        or does not include such a profile designator,
+        can be considered not to be a <a>DAPT Document</a>;
+        therefore a processor requiring strict adherence to DAPT could stop processing
+        such a document.</p>
+      </section>
+
+      <section>
+        <h3>Not supporting features excluded by the content profile</h3>
+        <p>A processor that takes as its input a <a>DAPT document</a> that
+          contains vocabulary relating to features that it does support,
+          but where support for those features is excluded from the content profiles
+          to which the document claims conformance, SHOULD NOT implement
+          those features in the context of that document.</p>
+      </section>
+      <section>
+        <h3>Handling <code>&lt;div&gt;</code> and <code>&lt;p&gt;</code> elements</h3>
+        <p>[[TTML2]] allows <code>&lt;div&gt;</code> elements to contain any combination of
+          <code>&lt;div&gt;</code> elements and <code>&lt;p&gt;</code> elements.
+          The DAPT data model describes how each <a>Script Event</a> is represented by
+          a <code>&lt;div&gt;</code> element that contains zero or more <code>&lt;p&gt;</code> elements.
+          It also permits other intermediate <code>&lt;div&gt;</code> elements in the path between
+          the <code>&lt;body&gt;</code> element and those <a>Script Event</a> <code>&lt;div&gt;</code> elements.
+          In addition, attributes not corresponding to properties in the DAPT data model are permitted.</p>
+        <p>This gives rise to possibilities such as:</p>
+        <ul>
+          <li><code>&lt;div&gt;</code> elements that contain children that are not <a>Text</a> objects,
+            for example other <code>&lt;div&gt;</code> elements or <code>&lt;metadata&gt;</code> elements;</li>
+          <li><code>&lt;div&gt;</code> elements that do not have attributes required for <a>Script Events</a>, like <code>xml:id</code>;</li>
+          <li><code>&lt;div&gt;</code> elements that have attributes absent from the DAPT data model for <a>Script Events</a>,
+            like <code>xml:space</code>;</li>
+          <li><code>&lt;p&gt;</code> elements whose parent <code>&lt;div&gt;</code> element does not appear to be a <a>Script Event</a>.</li>
+        </ul>
+
+        <p>The following processing rules resolve these cases.</p>
+        <p>Rules for identifying <a>Script Events</a>:</p>
+        <ol>
+          <li>A <code>&lt;div&gt;</code> element that has no <code>&lt;div&gt;</code> element children
```

Comment:
The reason we single out `div` elements here is because it is the only recognized vocabulary that is permitted by TTML2 in a `body` element and below, right?

Reply:
It's the only one permitted that is also mapped from the DAPT Data model, yes.

Reply:
No change needed for this review comment.